packages/vuetify/src/components/VDataTable/VDataTableHeaders.tsx

```diff
@@ -102,6 +110,9 @@ export const VDataTableHeaders = genericComponent<VDataTableHeadersSlots>()({
 
     const { backgroundColorClasses, backgroundColorStyles } = useBackgroundColor(props, 'color')
 
+    const { mobile } = useDisplay()
```

Comment:
Add display props https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/composables/display.ts#L216

Reply:
Pushed a change for this, let me know if I did this correctly and/or what you were looking for.